backend/sentiment/prototype/neuron.py

```diff
@@ -0,0 +1,45 @@
+from dataclasses import dataclass
+import numpy as np
+
+@dataclass
+class Neuron:
+    w: np.array  # weights
+    b: float = 0.0  # bias
+    learning_rate: float = 0.01  # learning rate for gradient descent
+    activation = None  # activation function, float
```

Comment:
though this hasn't been decided on yet it should still be set up with default functions, perhaps set a default if self.activation = none

Reply:
It is

Reply:
Sure. relu?

Reply:
forgot to say yes to this but sure i see it has already been done tho